src/mixins/I13nMixin.js

```diff
@@ -108,8 +109,16 @@ var I13nMixin = {
         self._i13nNode.setDOMNode(ReactDOM.findDOMNode(self));
 
         // enable viewport checking if enabled
-        if (self._getReactI13n().isViewportEnabled()) {
-            self.subscribeViewportEvents();
+        if (reactI13n.isViewportEnabled()) {
+            var options = {};
+            if ( reactI13n.getScrollableContainerId &&
```

Comment:
shouldn't need to check `reactI13n.getScrollableContainerId` and `reactI13n.getScrollableContainerDOMNode`, we can execute it directly